certgen/qrcode-image-generator/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target/
```

Comment:
You can possibly do away with file, if you add **/target and **/*.iml. Try doing it at project root.